spec/request_helper.rb

```diff
@@ -8,6 +8,12 @@ def self.stub_http_requests
 
     WebMock.stub_request(:get, "https://api.postcodes.io/postcodes/AA11AA")
            .to_return(status: 200, body: "{\"status\":200,\"result\":{\"postcode\":\"AA1 1AA\",\"admin_district\":\"Westminster\",\"codes\":{\"admin_district\":\"E09000033\"}}}", headers: {})
+    WebMock.stub_request(:get, "https://api.postcodes.io/postcodes/AA12AA")
+           .to_return(status: 200, body: "{\"status\":200,\"result\":{\"postcode\":\"AA1 2AA\",\"admin_district\":\"Westminster\",\"codes\":{\"admin_district\":\"E09000033\"}}}", headers: {})
+    WebMock.stub_request(:get, "https://api.postcodes.io/postcodes/NW1L5DP")
+           .to_return(status: 200, body: "{\"status\":200,\"result\":{\"postcode\":\"NW1L 5DP\",\"admin_district\":\"Westminster\",\"codes\":{\"admin_district\":\"E09000033\"}}}", headers: {})
+    WebMock.stub_request(:get, "https://api.postcodes.io/postcodes/ZZ11ZZ")
+           .to_return(status: 200, body: "{\"status\":200,\"result\":{\"postcode\":\"ZZ1 1ZZ\",\"admin_district\":\"Westminster\",\"codes\":{\"admin_district\":\"E09000033\"}}}", headers: {})
```

Comment:
These are just to make existing tests pass - should I add new tests for the edit local authority page or should that be a separate ticket?

Reply:
Have added tests to this ticket

Reply:
Wonder if it would be better to have these with the specific tests they're used in, since I guess they're not generic?